services/ui-src/src/components/pages/Export/ExportedReportPage.tsx

```diff
@@ -93,6 +76,119 @@ export const ExportedReportPage = () => {
   );
 };
 
+export const renderReportHeading = (
+  reportType: ReportType,
+  pageVerbiage: any,
+  report: ReportShape
+): JSX.Element => {
+  switch (reportType) {
+    case ReportType.MCPAR:
+    case ReportType.NAAR:
+      return (
+        <Heading as="h1" sx={sx.heading}>
+          {pageVerbiage.heading}
+          {report.fieldData.stateName}: {report.programName}
+        </Heading>
+      );
+    case ReportType.MLR:
+      return (
+        <Heading as="h1" sx={sx.heading}>
+          {report.fieldData.stateName}: {pageVerbiage.heading}
+        </Heading>
+      );
+    default:
+      assertExhaustive(reportType);
+      throw new Error(
+        `The heading for report type ${reportType} has not been implemented`
+      );
+  }
+};
+
+export const renderReportMetadataTables = (
+  reportType: ReportType,
+  pageVerbiage: any,
+  report: ReportShape
+): JSX.Element => {
+  switch (reportType) {
+    case ReportType.MCPAR:
+    case ReportType.NAAR:
+      return (
+        <Table
+          sx={sx.metadataTable}
+          content={{
+            headRow: [
+              pageVerbiage.metadataTableHeaders.dueDate,
+              pageVerbiage.metadataTableHeaders.lastEdited,
+              pageVerbiage.metadataTableHeaders.editedBy,
+              pageVerbiage.metadataTableHeaders.status,
+            ],
+            bodyRows: [
+              [
+                convertDateUtcToEt(report.dueDate),
+                convertDateUtcToEt(report.lastAltered),
+                report.lastAlteredBy,
+                report.status,
+              ],
+            ],
+          }}
+        />
+      );
+    case ReportType.MLR:
+      return (
+        <Table
+          sx={sx.metadataTable}
+          content={{
+            headRow: [
+              pageVerbiage.metadataTableHeaders.submissionName,
+              pageVerbiage.metadataTableHeaders.lastEdited,
+              pageVerbiage.metadataTableHeaders.editedBy,
+              pageVerbiage.metadataTableHeaders.status,
+            ],
+            bodyRows: [
+              [
+                report.submissionName ?? "",
+                convertDateUtcToEt(report.lastAltered),
+                report.lastAlteredBy,
+                report.status,
+              ],
+            ],
+          }}
+        />
+      );
+    default:
+      assertExhaustive(reportType);
```

Comment:
I like this alternative better than the assignment to `never`, thanks for adding!